### PR TITLE
[AWS] - Fix rolearn argument bug

### PR DESCRIPTION
**Pull request by GuyAfik**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-22618)

## Description
Fixed an issue where the *RoleArn* argument was always ignored when providing it through any command.
## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GuyAfik** [comment]
failed unit-tests: https://code.pan.run/xsoar/content/-/jobs/23606271

File: Packs/AWS-SecurityHub/ReleaseNotes/1_2_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### AWS - Security Hub
+
+- Fixed an issue where the *RoleArn* argument was always ignored when providing it through any command.
```
Comment (JudahSchwartz) [inline Packs/AWS-SecurityHub/ReleaseNotes/1_2_1.md:6]:
```suggestion
- Fixed an issue in all commands that accept the *RoleArn* argument, where the *RoleArn* argument was always ignored.
```

Comment (JudahSchwartz) [reply Packs/AWS-SecurityHub/ReleaseNotes/1_2_1.md:6]: And so for the rest of the RN

**Review by JudahSchwartz — APPROVED** [review]
nice change

File: Packs/AWS-SecurityHub/ReleaseNotes/1_2_1.md (hunk 1 above)
Comment (GuyAfik) [reply Packs/AWS-SecurityHub/ReleaseNotes/1_2_1.md:6]: done :)

**Pull request merged**